### PR TITLE
Updated decred-v5.css

### PR DESCRIPTION
**Pull request by vincent-bekir1**
Update the following:
* Increased the font size of the left pane.
* Reduced unused space on the large screen
* Changed the color of the Note on the left pane to match up with that on the home page.
* Changed the style of links over the whole page to make it more appealing.
* The text on the left and right pane are now top aligned

### CODE REVIEW

**Comment by dajohi** [comment]
@vincent-bekir1 please do not open new PRs but rather push more commits to the existing PR.

**Comment by vincent-bekir1** [comment]
> @vincent-bekir1 please do not open new PRs but rather push more commits to the existing PR.

Please pardon me, I only wanted to reset the old formatting. Which at the time of the commit, starting a new branch in another environment and finally new PR was the only option I could think of.

**Comment by peterzen** [comment]
@vincent-bekir1 What issue is this PR addressing?  (Please reference it in your commit message in order to make it straightforward to test - I can't seem to find the relevant issue ATM)

The changes in this PR are accessible on https://dcrweb.netlify.com/ - as this supposedly addresses design issue maybe @linnutee can verify that it's correct?

File: src/assets/css/decred-v5.css [hunk 1]
```diff
@@ -3751,7 +3751,6 @@ html[dir="rtl"] .lang-menu {
   -webkit-transition: all 400ms ease;
   transition: all 400ms ease;
 }
-
```
Comment (peterzen) [inline src/assets/css/decred-v5.css]: unneccessary white space

File: src/assets/css/decred-v5.css [hunk 2]
```diff
@@ -4091,18 +4090,15 @@ html[dir="rtl"] .lang-menu {
   -ms-flex-direction: row;
   flex-direction: row;
 }
-
```
Comment (peterzen) [inline src/assets/css/decred-v5.css]: unneccessary white space

File: src/assets/css/decred-v5.css [hunk 3]
```diff
@@ -4091,18 +4090,15 @@ html[dir="rtl"] .lang-menu {
...
   width: 40%;
 }
-	
 .community-section-right {
   -webkit-box-flex: 1;
   -webkit-flex: 1;
   -ms-flex: 1;
   flex: 1;
 }
-
```
Comment (peterzen) [inline src/assets/css/decred-v5.css]: unneccessary white space

File: src/assets/css/decred-v5.css [hunk 4]
```diff
@@ -4165,7 +4161,7 @@ html[dir="rtl"] .lang-menu {
 }
 
 .left-wrapper {
-  width: 320px;
```
Comment (peterzen) [inline src/assets/css/decred-v5.css]: This is a site wide change - I don't think this is correct, please reference the GH issue

File: src/assets/css/decred-v5.css [hunk 5]
```diff
@@ -7253,4 +7261,3 @@ p.get-into__link {
         width: calc(50vw - 160px);
     }
 }
-
```
Comment (peterzen) [inline src/assets/css/decred-v5.css]: unneccessary white space

**Pull request closed without merging**